### DISCUSSION TITLE
Mobile Scrolling Issue

### DISCUSSION THREAD

**Discussion by rubyswolf** [jcubic/jquery.terminal#880] [Q&A] [answered]
I'm on a nexus 6P android device and whenever I type a key on the onscreen keyboard the my terminal page scrolls down below all of the text to black and I can't see what I'm typing. I can scroll back up and then see the text again but if I try to type it goes back down. any ideas?

**Comment by jcubic** [comment, marked as answer]
This is the issue with the recent change in Google Chrome viewport sizes. This is the issue with all Android phones that use the latest Chrome.

You can fix the issue by adding meta tags to your HTML. This is explained in this article:

https://github.com/bramus/viewport-resize-behavior/blob/main/explainer.md

and

https://developer.chrome.com/blog/viewport-resize-behavior/


I wanted to fix the issue without the need to set up the mata tags in #707 but I think that the easiest will be to just set the meta tag.

So the solution is to set:

```html
<meta name="viewport" content="width=device-width, initial-scale=1.0, interactive-widget=resizes-content" />
```
